book/src/faq.md

```diff
@@ -96,20 +96,22 @@ when a change is signaled, primarily to minimize latency and optimize client req
 By default, if your service does not emit change notifications, the property values will not
 update accordingly. 
 
+
 However, you can disabling caching for specific properties:
 
 - Add the `#[dbus_proxy(property(emits_changed_signal = "false"))]` annotation to the property
-for which you desire to disable caching on.
+for which you desire to disable caching on. 
+
+> For more information about all the possible values for `emits_changed_signal` refer
+ to [`dbus_proxy`] documentation.
 
-- Use `ProxyBuilder` to build your proxy instance and use `ProxyBuilder::uncached_properties` method
+- Use `ProxyBuilder` to build your proxy instance and use [`ProxyBuilder::uncached_properties`](https://docs.rs/zbus/3/zbus/struct.ProxyBuilder.html#method.uncached_properties) method
```

Comment:
As I mentioned, please use the footnote-style linking. Also try your best to keep lines to < 100 chars.

Reply:
My bad, I did add one footnote style one because I assumed you were only asking for me to change that single one. I'll change all of my links to foot notes style